packages/generators/generators/lib/templates/ts/database.ts.hbs

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+/**
+ * {{ id }} database configuration
+ */
+
+
+module.exports = ({ env }) => ({
```

Comment:
I think if we default to anything it should be sqlite just like a new Strapi. But we may also want to have nothing set by default.

Reply:
nothing set is risky, I though to implement the question in the process of the evnironment generation

Reply:
I think if we have the time, adding the question could be nice yeah. Might be able to reuse some of the code/logic for the DB selection when generating a new project.